portfolio/src/main/webapp/contact.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Contact - Remus Neatu</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  
+  <body class="background">
+    <button onclick=goToContactPage()><b>Contact</b></button>
```

Comment:
If it's not part of exercise, It should be <a> element and not a button with a custom js logic because of different reasons:
It navigates to the different page so link is most appropriate.
Browther handles it differently e.g. open in a new tab option.

Reply:
Do you mean I should just have a text that, when clicked, redirects me to the respective page and not use the button format at all?